keps/sig-node/2413-seccomp-by-default/README.md

```diff
@@ -280,53 +298,48 @@ _This section must be completed when targeting beta graduation to a release._
 _This section must be completed when targeting beta graduation to a release._
 
 - **How can an operator determine if the feature is in use by workloads?**
-  Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
-  checking if there are objects with field X set) may be a last resort. Avoid
-  logs or events for this purpose.
+
+  If a new workload gets created and no `securityContext.seccompProfile.type` is
+  being specified, then this field will default to `RuntimeDefault` instead of
+  `Unconfined`.
 
 - **What are the SLIs (Service Level Indicators) an operator can use to determine
   the health of the service?**
 
-  - [ ] Metrics
-    - Metric name:
-    - [Optional] Aggregation method:
-    - Components exposing the metric:
-  - [ ] Other (treat as last resort)
-    - Details:
+  - A workload is exiting unexpectedly after the feature has been enabled.
+
+    - The termination reason is a "permission denied" error.
+    - The termination is reproducible.
+    - Replacing `SCMP_ACT_ERRNO` to `SCMP_ACT_LOG` in the default profile will
+      show seccomp error messages in auditd or syslog.
+    - There are no other reasons for container termination (like eviction or
+      exhausting resources)
+
+  - A workload is not behaving completely functional, for example some features
+    are misbehaving but the appliction does not exit.
+
+    - There are permission denied errors in the workload logs.
+    - The behavior is reproducible.
+    - Replacing `SCMP_ACT_ERRNO` to `SCMP_ACT_LOG` in the default profile will
+      show seccomp error messages in auditd or syslog.
 
 - **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
-  At a high level, this usually will be in the form of "high percentile of SLI
-  per day <= X". It's impossible to provide comprehensive guidance, but at the very
-  high level (needs more precise definitions) those may be things like:
 
-  - per-day percentage of API calls finishing with 5XX errors <= 1%
-  - 99% percentile over day of absolute value from (job creation time minus expected
-    job creation time) for cron job <= 10%
-  - 99,9% of /health requests per day finish with 200 code
+  - The workload availability, functionality and health is exactly the same with
```

Comment:
Is there a metric that tracks pod crashloops?  That would be enough to allow a cluster-admin to notice a problem in aggregate, right?

Reply:
Yep, changed the section to mention that.